ydb/library/yql/yt/dynamic/impl.cpp

```diff
@@ -75,7 +75,8 @@ TBridgeQueryResult* BridgeRun(
     const char* queryText,
     const char* settings,
     const TBridgeQueryFile* bridgeFiles,
-    int bridgeFileCount)
+    int bridgeFileCount,
+    int executeMode)
```

Comment:
You should also modify the bridge file
https://github.com/ydb-platform/ydb/blob/bbb26a84956dd242b2e8f6947e980f69b3b09035/ydb/library/yql/yt/bridge/interface.h#L93C24-L93C42